content/en/docs/ops/integrations/spire/index.md

```diff
@@ -0,0 +1,364 @@
+---
+title: Istio CA Integration with SPIRE 
+linktitle: How to integrate Istio with SPIRE through Envoy SDS API.
+description: Describes how to configure Istio to integrate with SPIRE to get cryptographic identities through Envoy's SDS API.
+weight: 40
+keywords: [kubernetes,spiffe,spire]
+aliases:
+owner: istio/wg-networking-maintainers
+test: no
+---
+
+[SPIRE](https://spiffe.io/docs/latest/spire-about/spire-concepts/) is a production-ready implementation of the SPIFFE specification that performs node and workload attestation in order to securely issue cryptographic identities to workloads running in heterogeneous environments.  
+SPIRE can be configured as a source of cryptographic identities for Istio workloads through an integration with [Envoy's SDS API](https://www.envoyproxy.io/docs/envoy/latest/configuration/security/secret).
+Istio can detect the existence of a UNIX domain socket that implements the Envoy SDS API on a defined socket path, allowing Envoy to communicate and fetch identities directly from it.
+
+This integration with SPIRE provides flexible attestation options not available with the default Istio identity management while harnessing Istio's powerful service management.
+For example, SPIRE's plugin architecture enables diverse workload attestation options beyond the Kubernetes namespace and service account attestation offered by Istio.
+SPIRE's node attestation extends attestation to the physical or virtual hardware on which workloads run.
+
+## Install SPIRE

```

Comment:
I'd like to add one more brief paragraph before "Install SPIRE" . The new proposed paragraph refers to the README.md doc written by @jufantozzi . I think it's good to have each doc (README.md and index.md) contain a link to the other doc.

For a quick demo of how this SPIRE integration with Istio works, see [Integrating SPIRE as a CA through Envoy's SDS API](FIXME-REAL-URL-TO-README.MD).

Reply:
Agreed, I believe it would be useful to the user 👍🏻

Reply:
+1 here